pallets/rmrk-core/src/lib.rs

```diff
@@ -410,13 +415,6 @@ pub mod pallet {
 				transferable,
 			)?;
 
-			pallet_uniques::Pallet::<T>::do_mint(
```

Comment:
Why is it removed?

Reply:
Moved these to the internal functions so we can call the internal functions & ensure that the uniques pallet still calls their `do_create_collection` & `do_mint`

Reply:
Oh sorry. I missed that part. However should we also move the event to the internal functions?